stream/management/commands/pittgoogle.py

```diff
@@ -0,0 +1,79 @@
+#!/usr/bin/env python3
+# -*- coding: UTF-8 -*-
+"""Listen to a Pitt-Google Pub/Sub stream and save alerts to the database."""
+from importlib import import_module
+import logging
+
+from django.conf import settings
+from django.core.management.base import BaseCommand
+from django.db import transaction
+
+from stream.models import Alert, Topic
+
+from pittgoogle import Consumer
```

Comment:
This `pittgoogle.Consumer` will be nearly identical to the `ConsumerStreamPython` class I wrote for the TOM Toolkit. 

Note that it is a fairly generic consumer that does not actually rely on the TOM Toolkit. Therefore I will probably move it to a separate repo of Pitt-Google tools. This will be analogous to Fink's `fink_client.consumer.AlertConsumer`, which I see you are importing in your `fink.py` listener.

Reply:
I modified the fink.py listener in branch `origin/u/johannct/streams` : this version does work and consumes live streams. I did not manage to get the `master` version work because the raw alert from fink has elements currently not properly taken care of (image cutouts).